knative-operator/pkg/common/openshift_test.go

```diff
@@ -116,3 +116,9 @@ func verifyCerts(t *testing.T, ks *servingv1alpha1.KnativeServing) {
 		t.Error("Missing custom certs config")
 	}
 }
+
+func verifyTimestamp(t *testing.T, ks *servingv1alpha1.KnativeServing) {
```

Comment:
Merge error? This shouldn't be needed anymore.

Reply:
Yep removed